src/frontend/apps/conversations/src/cunningham/cunningham-style.css

```
@@ -1,3 +1,4 @@
+@import url('@gouvfr-lasuite/cunningham-react/style');
```

Stylelint (17.10.0)

[error] Expected "url('@gouvfr-lasuite/cunningham-react/style')" to be "'@gouvfr-lasuite/cunningham-react/style'" (import-notation)

(import-notation)